### ISSUE TITLE
What factor make the metric of mIOU higher than paper write?

### ISSUE THREAD

**Issue by zhijiejia** [yassouali/pytorch_segmentation#99]
Thank for you excellent work
I notice that the mIOU can up to 82%, but the author of PSPNet only acheive 76% - 77% ues resnet50 as backbone, what make the difference ?  Can you descirbe it ? Thank you so much.

**Comment by zhijiejia** [comment]
Do you modify some network structure to improve the performance？

**Comment by yassouali** [comment]
Hi  @zhijiejia 

If you are talking about Pascal VOC 2012, I just checked the paper and the results seem to be similar as found in Table 6 of the paper (note that their results is the test set and not val set, so 82 might translate to ~ 80 / 81 in the test set evaluation done in the evaluation server)

**Comment by zhijiejia** [comment]
sorry, i see the result(76% - 77%) in author's pspnet project, you are right,

**Comment by zhijiejia** [comment]
By the way,  do you use the default config in this project when you train the pascal voc2012 dataset get the 82% mIOU on val set, , I run the default config, get the result: epoch: 80, mIOU on val set: 0.7739

**Comment by yassouali** [comment]
Hi @zhijiejia 

It has been a while since I run a test, so I don't quite remember the exact value. If you have a better config I would love to update it, thank.

**Comment by zhijiejia** [comment]
Ok, Thank you for your prompt reply

**Comment by zhijiejia** [comment]
Oh, I read the paper just，I find the paper get mIOU: 82，author use the resnet101 as its backbone. So there is reasonable for mIOU: 78% for resnet50 as backbone if we don't find a better way to train PSPNet.
